server/db/interface.go

```diff
@@ -52,6 +52,13 @@ type EpochResults struct {
 	OrdersMissed      []order.OrderID
 }
 
+// OrderStatus is the current status of an order, including its fill.
+type OrderStatus struct {
+	ID     order.OrderID
+	Status order.OrderStatus
+	Fill   int64
```

Comment:
I'll be interested to see if `Fill` is actually useful. Did you have a need for this already?

Reply:
Not something we can't do without, but I currently use this to update the filled value on the client when I use the order_status route. The screenshots and description in https://github.com/decred/dcrdex/pull/677#issuecomment-693781823 show how this has been handy.